.github/workflows/nightly.yml

```diff
@@ -0,0 +1,31 @@
+
+name: Nightly Release
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
+jobs:
+  nightly:
+    name: Nightly Release
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout repo"
+        uses: actions/checkout@v2
+
+      - name: Deploy using Release Action
+        uses: ncipollo/release-action@v1
+        with:
+          name: Latest Nightly
+          tag: nightly
+          commit: v3.4/dev
```

Comment:
We should add to [Release-Procedure Wiki page](https://github.com/coreruleset/coreruleset/wiki/Release-Procedure) to modify this.